### DISCUSSION TITLE
Higher quality download

### DISCUSSION THREAD

**Discussion by theb0y1** [pear-devs/pear-desktop#657] [Ideas]
Maybe adding a drop down menu in the downloader part to download higher kbps mp3 files if possible

**Comment by TommyHrablo** [comment]
> Maybe adding a drop down menu in the downloader part to download higher kbps mp3 files if possible

Especially if you are logged in via google account and you have a YT or YT music premium, right?

**Comment by TommyHrablo** [comment]
> Maybe adding a drop down menu in the downloader part to download higher kbps mp3 files if possible

According to this topic:
https://github.com/th-ch/youtube-music/issues/112
there is a way, but you need to "manually edit config.json". I do not understand how to configure properly... (like I would like to download to MP3 with a 320kbps.
It would be great to see more predefined "Presets options" in download plug-in when enabled. That would be a very nice and useful for people like me and also as well if there is a way how you can benefit if you are own YT or YT music Premium - If is possible according to used API.

**Comment by freshSolero** [comment]
How I can I change the presets? I found the types.ts file here where the FFMPEG instructions are defined but I cannot find it on my Mac. It would be great just to have it in a different container as webm is difficult to handle.
